wrapper/Ada/alire.toml

```diff
@@ -0,0 +1,15 @@
+name = "wolfssl_ada"
```

Comment:
This is being nitpicky, but, is the `_ada` really necessary here? By virtue of being an Alire crate, the `ada` part is implied.

Reply:
Makes sense. I will take out the `_ada` surname :)